### PR TITLE
globoplay: big brother n' stuff

### PR DESCRIPTION
**Pull request by fr0r**
Big Brother is back for another season!

This means i need to ~~spam the shit out of ``if else``~~ add the 12 new cameras they've added + some random radio station.

<details>

<summary><b>changelog</b> <i>(click to expand)</i></summary>

- Changed ownership of the presence since i'm doing most of the things.
- Added Big Brother cameras.
- Added local CBN (all-news radio) stations.
- Shortened some queries

</details>

<details>

<summary><b>screenshots</b> <i>(click to expand)</i></summary>

image

image

</details>

### CODE REVIEW

**Comment by fr0r** [comment]
there we go lol @ririxidev

File: websites/G/globoplay/dist/metadata.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "author": {
-    "name": "Bas950",
-    "id": "241278257335500811"
+    "name": "fror",
+    "id": "515668127829458945"
```
Comment (ririxidev) [inline websites/G/globoplay/dist/metadata.json:4]: bas accepted owner change?

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/G/globoplay/dist/metadata.json (hunk 1 above)
Comment (fr0r) [reply websites/G/globoplay/dist/metadata.json:4]: yup
image

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
bump version

**Review by Bas950 — APPROVED** [review]
bump version

**Review by ririxidev — APPROVED** [review]

**Pull request merged**